anyway/infographics_utils.py

```diff
@@ -61,768 +144,678 @@ def serialize(self):
-    total_count = 0
-    for item in involved_by_vehicle_type_data:
-        vehicle_type, count = item["involve_vehicle_type"], int(item["count"])
-        total_count += count
-        if vehicle_type in BE_CONST.CAR_VEHICLE_TYPES:
-            driver_types["רכב פרטי"] += count
-        elif vehicle_type in BE_CONST.LARGE_VEHICLE_TYPES:
-            driver_types["מסחרי/משאית"] += count
-        elif vehicle_type in BE_CONST.MOTORCYCLE_VEHICLE_TYPES:
-            driver_types["אופנוע"] += count
-        elif vehicle_type in BE_CONST.BICYCLE_AND_SMALL_MOTOR_VEHICLE_TYPES:
-            driver_types["אופניים/קורקינט"] += count
-        else:
-            driver_types["אחר"] += count
-
-    output = defaultdict(float)
-    for k, v in driver_types.items():  # Calculate percentage
-        output[k] = 100 * v / total_count
-
-    return output
-
-
-@lru_cache(maxsize=64)
-def percentage_accidents_by_car_type_national_data_cache(start_time, end_time):
-    involved_by_vehicle_type_data = get_accidents_stats(
-        table_obj=InvolvedMarkerView,
-        group_by="involve_vehicle_type",
-        count="involve_vehicle_type",
-        start_time=start_time,
-        end_time=end_time,
-    )
-    return percentage_accidents_by_car_type(involved_by_vehicle_type_data)
-
-
-def stats_accidents_by_car_type_with_national_data(
-    involved_by_vehicle_type_data, start_time, end_time
-):
-    out = []
-    data_by_segment = percentage_accidents_by_car_type(involved_by_vehicle_type_data)
-    national_data = percentage_accidents_by_car_type_national_data_cache(start_time, end_time)
-
-    for k, v in national_data.items():  # pylint: disable=W0612
-        out.append(
-            {
-                "car_type": k,
-                "percentage_segment": data_by_segment[k],
-                "percentage_country": national_data[k],
-            }
+            # Parse the db age range
+            # noinspection RegExpRedundantEscape
+            match_parsing = re.match("([0-9]{2})\\-([0-9]{2})", age_range)
+            if match_parsing:
+                regex_age_matches = match_parsing.groups()
+                if len(regex_age_matches) != 2:
+                    dict_by_required_age_group["unknown"] += count
+                    continue
+                min_age_raw, max_age_raw = regex_age_matches
+            else:
+                match_parsing = re.match("([0-9]{2})\\+", age_range)  # e.g  85+
+                if match_parsing:
+                    # We assume that no body live beyond age 200
+                    min_age_raw, max_age_raw = match_parsing.group(1), 200
+                else:
+                    dict_by_required_age_group["unknown"] += count
+                    continue
+
+            # Find to what "bucket" to aggregate the data
+            min_age = int(min_age_raw)
+            max_age = int(max_age_raw)
+            for item in range_dict.items():
+                item_min_range, item_max_range = item
```

Comment:
You can do `tem_min_range, item_max_range in range_dict.items():`

Reply:
Done